pkg/api/types.go

```diff
@@ -367,6 +373,28 @@ type GCEPersistentDiskVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// AWSPersistentDiskVolumeSource represents a Persistent Disk resource in AWS.
+//
+// An AWS EBS disk must exist and be formatted before mounting to a container.
+// The disk must also be in the same AWS zone as the kubelet.
+// A AWS EBS disk can only be mounted as read/write once.
+type AWSPersistentDiskVolumeSource struct {
+	// Unique name of the PD resource. Used to identify the disk in AWS
+	PDName string `json:"pdName"`
```

Comment:
Fair enough!  Making the change now.  VolumeId feels like a better name even for GCE :-)